vrk-kpa/suomidigi
# SUOPA-904 Print styles

To test: 
- `make fresh`
- Open article, blog post, core content, basic page and a legislation card in new tabs (links below). Check that the Print this page -button exists in the bottom of the page. **You might need to do hard reload for the browser to refresh the icons file**
   - Click on the button and check that it triggers print preview dialog.
   - Check from the print preview that there is no navigation, sidebar or footer visible.
- Links to content types:
   - Legislation card: https://suomidigi.docker.sh/node/139
   - Core content: https://suomidigi.docker.sh/node/54 
   - Article: https://suomidigi.docker.sh/node/792
   - Blog post: https://suomidigi.docker.sh/node/165
   - Basic page: https://suomidigi.docker.sh/node/727
